src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -27,9 +27,16 @@ protocol ExposureSummaryProvider: AnyObject {
 }
 
 final class RiskProvider {
-	private let consumers = NSHashTable<RiskConsumer>.weakObjects()
-	private let queue = DispatchQueue(label: "com.sap.RiskLevelProvider")
+
+	private let queue = DispatchQueue(label: "com.sap.riskprovider.queue")
```

Comment:
```suggestion
	private let queue = DispatchQueue(label: "com.sap.RiskLevelProvider")
```

`.queue` is redundant and the naming convention usually is `$prefix.$className/structName`.

Reply:
ok, I reverted labels